dll/ntdll/nt_0600/DllMain.c

```diff
@@ -1,15 +1,26 @@
 #include "ntdll_vista.h"
 
+NTSTATUS
+NTAPI
+RtlpInitializeLocaleTable(VOID);
+
 BOOL
 WINAPI
 DllMain(HANDLE hDll,
         DWORD dwReason,
         LPVOID lpReserved)
 {
+    NTSTATUS Status;
+
     if (dwReason == DLL_PROCESS_ATTACH)
     {
         LdrDisableThreadCalloutsForDll(hDll);
         RtlpInitializeKeyedEvent();
+        Status = RtlpInitializeLocaleTable();
+        if (!NT_SUCCESS(Status))
+        {
+            return FALSE;
```

Comment:
Should the keyed event be cleaned up here?

Reply:
Yeah, that is probably a good idea.